### ISSUE TITLE
Character encoding in French catalogue

### ISSUE THREAD

**Issue by zuphilip** [zotero/translators#848]
The character encoding in these two testcases from "Library Catalog (Koha)" translator are wrong:

https://ahuntsic.koha.ccsr.qc.ca/cgi-bin/koha/opac-detail.pl?biblionumber=71166
https://ahuntsic.koha.ccsr.qc.ca/cgi-bin/koha/opac-detail.pl?biblionumber=145770

Moreover, there seem to be some other differences. @aurimasv : Is this connected to your fix in the character normalization?

**Comment by adam3smith** [comment]
no, that's likely a Koha issue - if not set-up correctly, Koha serves MARC with an incorrect content header (couple of threads on this on the forums). We can try contacting the admin, might work.
